opacclient/libopac/src/main/java/de/geeksfactory/opacclient/objects/AccountItem.java

```diff
@@ -0,0 +1,148 @@
+package de.geeksfactory.opacclient.objects;
+
+public abstract class AccountItem {
+    private long account;
+    private String title;
+    private String author;
+    private String format;
```

Comment:
Was there a reason for having a string here instead of moving ``SearchResult.MediaType`` to e.g. ``OpacApi`` and using it in both places? This would enable us to show icons in the acocunt view in the future. String serialiazation for the database should be done at database communication time.